vimish-fold.el

```diff
@@ -311,6 +338,15 @@ If OVERLAY does not represent a fold, it's ignored."
    #'vimish-fold--vimish-overlay-p
    (overlays-in beg end)))
 
+(defun vimish-fold--folds-beginning-in (beg end)
+  (let ((list (vimish-fold--folds-in beg end)) tmp-ov)
+    (while (list)
+      (setq tmp-ov (car list))
+      (unless (and (>= pos (overlay-start tmp-ov))
```

Comment:
I get the following warning here:

> reference to free variable `pos`

You should first declare the variable using `let` and then use it.

Reply:
Woops it should have been beg and end, how did this even work ? ><